action.yml

```diff
@@ -55,12 +55,12 @@ outputs:
     description: "number of mabl plans that passed against this deployment.  A mabl plan is a collection of similarly configured tests."
   plans_failed:
     description: "number of mabl plans that failed against this deployment.  A mabl plan is a collection of similarly configured tests."
-  journeys_run:
-    description: "total number of mabl journeys run against this deployment.  A mabl journey is an end to end test of your application."
-  journeys_passed:
-    description: "number of mabl journeys that passed against this deployment.  A mabl journey is an end to end test of your application."
-  journeys_failed:
-    description: "number of mabl journeys that failed against this deployment.  A mabl journey is an end to end test of your application."
+  tests_run:
+    description: "total number of mabl tests run against this deployment.  A mabl tests is an end to end test of your application."
```

Comment:
Shouldn't this part about end to end also be removed like in the ``README.md`?

Reply:
Good catch.  I'll uodate